articles/sentinel/authentication-normalization-schema.md

```diff
@@ -38,7 +38,8 @@ For more information about normalization in Azure Sentinel, see [Normalization a
 
 Azure Sentinel provides the following built-in, product-specific authentication event parsers: 
 
-- **Security Events sign-ins (Events 4624 and 4625)**, collected using the Log Analytics Agent or Azure Monitor Agent
+- **Windows sign-ins** reported as Security Events (4624, 4625, 4634, and 4647), collected using the Log Analytics Agent or Azure Monitor Agent
```

Comment:
@oshezaf missing period, defender for endpoint (singular):

- **Windows sign-ins** reported as Security Events (4624, 4625, 4634, and 4647), collected using the Log Analytics Agent or Azure Monitor Agent.
- **Windows sign-ins** reported by Microsoft 365 Defender for Endpoint, collected using the Microsoft 365 Defender connector.